src/Relation/Binary/HeterogeneousEquality.agda

```diff
@@ -79,6 +79,10 @@ subst-removable : ∀ (P : Pred A p) {x y} (eq : x ≅ y) z →
                   subst P eq z ≅ z
 subst-removable P refl z = refl
 
+subst₂-removable : ∀ (_∼_ : REL A B ℓ) {x y u v} (eq₁ : x ≅ y) (eq₂ : u ≅ v) p →
```

Comment:
It would help understanding if you could add the type of `p` here! (and while we're at it, add the type of `z` in the lemmas above and below + perhaps standardise the name).